### PR TITLE
Add more missing commands

### PR DESCRIPTION
**Pull request by archseer**
flynn kill, flynn server-remove, flynn servers.

### CODE REVIEW

File: kill.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+var cmdKill = &Command{
+  Run:   runKill,
+  Usage: "kill <job>",
+  Short: "kill a job",
+  Long:  `Kill a job`,
+}
+
+func runKill(cmd *Command, args []string, client *controller.Client) error {
+  if len(args) == 0 {
```
Comment (titanous) [inline kill.go]: This should be `if len(args) != 1`

File: kill.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    cmd.printUsage(true)
+  }
+
+  job := args[0]
+
+  if err := client.DeleteJob(mustApp(), job); err != nil {
+    return err
+  }
+  log.Printf("Key %s removed.", job)
```
Comment (titanous) [inline kill.go]: Copypasta.

File: server.go [hunk 3]
```diff
@@ -11,11 +11,39 @@ import (
...
+var cmdServers = &Command{
+	Run:      runServers,
+	Usage:    "servers",
+	Short:    "list servers",
+	Long:     `List all servers in the ~/.flynnrc configuration file`,
+	NoClient: true,
+}
+
+func runServers(cmd *Command, args []string, client *controller.Client) error {
+	if len(args) == 1 {
```
Comment (titanous) [inline server.go]:
``` go
if len(args) != 0 {
```

File: kill.go (hunk 2 above)
Comment (archseer) [inline kill.go]: Yes indeed, caught redhanded.

File: server.go [hunk 4]
```diff
@@ -11,11 +11,39 @@ import (
...
+	if err := readConfig(); err != nil {
+		return err
+	}
+
+	w := tabWriter()
+	defer w.Flush()
+
+	listRec(w, "NAME", "URL", "KEY")
+	for _, s := range config.Servers {
+		listRec(w, s.Name, s.URL, s.Key)
```
Comment (titanous) [inline server.go]: Don't print the key.

File: server.go [hunk 5]
```diff
@@ -81,3 +109,44 @@ func runServerAdd(cmd *Command, args []string, client *controller.Client) error
...
+		cmd.printUsage(true)
+	}
+	if err := readConfig(); err != nil {
+		return err
+	}
+
+	name := args[0]
+
+	for i, s := range config.Servers {
+		if s.Name == name {
```
Comment (titanous) [inline server.go]: Invert this:

``` go
if s.Name != name {
    continue
}
```

**Comment by titanous** [comment]
Closing, as this was superseded by #18.

**Pull request closed without merging**